backend-rust/src/indexer.rs

```diff
@@ -542,6 +561,9 @@ impl ProcessEvent for BlockProcessor {
             for item in block.prepared_block_items.iter() {
                 item.save(&mut tx).await?;
             }
+            for item in block.special_items.iter() {
```

Comment:
We could maybe use a trait here, but since we do not do that for events either, I am fine with this. Also the benefit here is rather small